tests/phpunit/README.md

```diff
@@ -0,0 +1,33 @@
+# PHPUnit Tests
+
+PHPUnit is the official testing framework chosen by the core team to test our PHP code.
+
+This documentation will assume you have a local development environment of your own choosing. For other testing workflows, such as Docker, see the [PHP: PHP Unit](https://make.wordpress.org/core/handbook/testing/automated-testing/phpunit/) handbook page.
+
+## Setup
+
+1. Check out the wordpress-develop repository
+
+       git clone git://develop.git.wordpress.org/ wordpress-develop
+       cd wordpress-develop
```

Comment:
```suggestion
cd wordpress-develop
```
For this and the `git clone` line above it, please wrap with three backticks instead of spaces ([for example](https://github.com/WordPress/wordpress-develop/blob/552077a27589294b2cf6097205a4045585016281/README.md?plain=1#L50-L55)).